README.md

```diff
@@ -22,7 +32,109 @@ The generated files will be placed in `the-output-directory-sys`. You now have t
 
 You'll now have to write another GIR file (take a look to [gtk/Gir.toml](https://github.com/gtk-rs/gtk/blob/master/Gir.toml) for an example).
 
-*This part needs more information about this toml file*
+### TOML file
+
+At the opposite of the FFI mode, this one only generates the specified objects. You can either add the object's fullname to the `generate` array or add it to the `manual` array (but in this case, you'll only be able to use the `sys` object). Example:
+
```

Comment:
Don't know how to say it: if object added to `manual` it will be used in generated code instead marking it "/*Ignored*/"
So manual only meant for other generated